en/ajax.texy

```diff
@@ -189,28 +188,6 @@ You can also combine it with dynamic snippets.
 Adding and deleting
 ===================
 
-If you add a new item into the list and invalidate `itemsContainer`, the AJAX request returns snippets including the new one, but the javascript handler won’t be able to render it. This is because there is no HTML element with the newly created ID.
+If you add a new item into the list and invalidate `itemsContainer`, the AJAX request returns snippets including the new one, but you need to make sure your client-side code can create the appropriate HTML element for it when the response is being handled. Currently only Nittro [can do that | https://github.com/nittro/nittro/wiki/The-Page-service#dynamic-snippets] out of the box, but it shouldn't be too hard with nette.ajax.js either.
```

Comment:
Nejsem si uplne jisty, ze tohle je spravne.

1. Podbizi to nittro. Ikdyz je nittro fajn, asi bychom meli byt neutralni.
2. Jak dlouho ten odkaz bude existovat? Co kdyz se smaze.. :-)